### ISSUE TITLE
[BUG] mfs.cgi - chunkserver load @ Servers tab wrong/too low

### ISSUE THREAD

**Issue by oszafraniec** [moosefs/moosefs#458]
## System information
Ubuntu (mix of 16, 18, 20)
Packaged by distro
MFS 3.0.116 pro and community

After upgrade from 3.0.115 to 3.0.116 I've noticed significant drop in load of chunkservers shown on Servers Tab. But in the Charts load is still shown like it used to be with 3.0.115 version. Traffic on the cluster is the same as it was or even increased over time since upgrade. I believe the Charts data is correct, Servers Tab is not.

We have 2 clusters, 1 pro, 1 community, same problem. We monitor them using 1 mfs.cgi, also upgrade'ed to 3.0.116 (python 3.6).

**Comment by chogata** [comment]
Actually, what is not correct, is the name of the chart and we should probably change that :)

The server tab shows current load, as it was in the moment the master was asked about it. The chart shows the maximum load from any given minute. So, if during one minute the load peaks to, let's say, 30, the chart will show 30. The server tab might show 30 if you refresh it at the right moment, or if the load persists. If it is just a very short peak, most of the time you will not catch it, so the server tab will show something lower.

Basically, the server tab is meant to show problems with chunkservers. If you don't see high loads there often, that means there are no problems (at least with load ;) ). The behaviour of 3.0.115 (when the server tab also showed the minute's peak) was actually more misleading, because one momentary peak was presented to the user as a load persisting for a full minute.

The chart shows history, but as I said, the name of the chart is misleading. We will change it and maybe add another chart showing medium load instead of peak load.

**Comment by oszafraniec** [comment]
@chogata now I get it and it makes sens :)
